lib/streamlit/emojis.py

```diff
@@ -23,7 +23,7 @@
, "🧖🏾‍♂️", "🧖🏿", "🧖🏿‍♀", "🧖🏿‍♀️", "🧖🏿‍♂", "🧖🏿‍♂️", "🧗", "🧗‍♀", "🧗‍♀️", "🧗‍♂", "🧗‍♂️", "🧗🏻", "🧗🏻‍♀", "🧗🏻‍♀️", "🧗🏻‍♂", "🧗🏻‍♂️", "🧗🏼", "🧗🏼‍♀", "🧗🏼‍♀️", "🧗🏼‍♂", "🧗🏼‍♂️", "🧗🏽", "🧗🏽‍♀", "🧗🏽‍♀️", "🧗🏽‍♂", "🧗🏽‍♂️", "🧗🏾", "🧗🏾‍♀", "🧗🏾‍♀️", "🧗🏾‍♂", "🧗🏾‍♂️", "🧗🏿", "🧗🏿‍♀", "🧗🏿‍♀️", "🧗🏿‍♂", "🧗🏿‍♂️", "🧘", "🧘‍♀", "🧘‍♀️", "🧘‍♂", "🧘‍♂️", "🧘🏻", "🧘🏻‍♀", "🧘🏻‍♀️", "🧘🏻‍♂", "🧘🏻‍♂️", "🧘🏼", "🧘🏼‍♀", "🧘🏼‍♀️", "🧘🏼‍♂", "🧘🏼‍♂️", "🧘🏽", "🧘🏽‍♀", "🧘🏽‍♀️", "🧘🏽‍♂", "🧘🏽‍♂️", "🧘🏾", "🧘🏾‍♀", "🧘🏾‍♀️", "🧘🏾‍♂", "🧘🏾‍♂️", "🧘🏿", "🧘🏿‍♀", "🧘🏿‍♀️", "🧘🏿‍♂", "🧘🏿‍♂️", "🧙", "🧙‍♀", "🧙‍♀️", "🧙‍♂", "🧙‍♂️", "🧙🏻", "🧙🏻‍♀", "🧙🏻‍♀️", "🧙🏻‍♂", "🧙🏻‍♂️", "🧙🏼", "🧙🏼‍♀", "🧙🏼‍♀️", "🧙🏼‍♂", "🧙🏼‍♂️", "🧙🏽", "🧙🏽‍♀", "🧙🏽‍♀️", "🧙🏽‍♂", "🧙🏽‍♂️", "🧙🏾", "🧙🏾‍♀", "🧙🏾‍♀️", "🧙🏾‍♂", "🧙🏾‍♂️", "🧙🏿", "🧙🏿‍♀", "🧙🏿‍♀️", "🧙🏿‍♂", "🧙🏿‍♂️", "🧚", "🧚‍♀", "🧚‍♀️", "🧚‍♂", "🧚‍♂️", "🧚🏻", "🧚🏻‍♀", "🧚🏻‍♀️", "🧚🏻‍♂", "🧚🏻‍♂️", "🧚🏼", "🧚🏼‍♀", "🧚🏼‍♀️", "🧚🏼‍♂", "🧚🏼‍♂️", "🧚🏽", "🧚🏽‍♀", "🧚🏽‍♀️", "🧚🏽‍♂", "🧚🏽‍♂️", "🧚🏾", "🧚🏾‍♀", "🧚🏾‍♀️", "🧚🏾‍♂", "🧚🏾‍♂️", "🧚🏿", "🧚🏿‍♀", "🧚🏿‍♀️", "🧚🏿‍♂", "🧚🏿‍♂️", "🧛", "🧛‍♀", "🧛‍♀️", "🧛‍♂", "🧛‍♂️", "🧛🏻", "🧛🏻‍♀", "🧛🏻‍♀️", "🧛🏻‍♂", "🧛🏻‍♂️", "🧛🏼", "🧛🏼‍♀", "🧛🏼‍♀️", "🧛🏼‍♂", "🧛🏼‍♂️", "🧛🏽", "🧛🏽‍♀", "🧛🏽‍♀️", "🧛🏽‍♂", "🧛🏽‍♂️", "🧛🏾", "🧛🏾‍♀", "🧛🏾‍♀️", "🧛🏾‍♂", "🧛🏾‍♂️", "🧛🏿", "🧛🏿‍♀", "🧛🏿‍♀️", "🧛🏿‍♂", "🧛🏿‍♂️", "🧜", "🧜‍♀", "🧜‍♀️", "🧜‍♂", "🧜‍♂️", "🧜🏻", "🧜🏻‍♀", "🧜🏻‍♀️", "🧜🏻‍♂", "🧜🏻‍♂️", "🧜🏼", "🧜🏼‍♀", "🧜🏼‍♀️", "🧜🏼‍♂", "🧜🏼‍♂️", "🧜🏽", "🧜🏽‍♀", "🧜🏽‍♀️", "🧜🏽‍♂", "🧜🏽‍♂️", "🧜🏾", "🧜🏾‍♀", "🧜🏾‍♀️", "🧜🏾‍♂", "🧜🏾‍♂️", "🧜🏿", "🧜🏿‍♀", "🧜🏿‍♀️", "🧜🏿‍♂", "🧜🏿‍♂️", "🧝", "🧝‍♀", "🧝‍♀️", "🧝‍♂", "🧝‍♂️", "🧝🏻", "🧝🏻‍♀", "🧝🏻‍♀️", "🧝🏻‍♂", "🧝🏻‍♂️", "🧝🏼", "🧝🏼‍♀", "🧝🏼‍♀️", "🧝🏼‍♂", "🧝🏼‍♂️", "🧝🏽", "🧝🏽‍♀", "🧝🏽‍♀️", "🧝🏽‍♂", "🧝🏽‍♂️", "🧝🏾", "🧝🏾‍♀", "🧝🏾‍♀️", "🧝🏾‍♂", "🧝🏾‍♂️", "🧝🏿", "🧝🏿‍♀", "🧝🏿‍♀️", "🧝🏿‍♂", "🧝🏿‍♂️", "🧞", "🧞‍♀", "🧞‍♀️", "🧞‍♂", "🧞‍♂️", "🧟", "🧟‍♀", "🧟‍♀️", "🧟‍♂", "🧟‍♂️", "🧠", "🧡", "🧢", "🧣", "🧤", "🧥", "🧦", "🧧", "🧨", "🧩", "🧪", "🧫", "🧬", "🧭", "🧮", "🧯", "🧰", "🧱", "🧲", "🧳", "🧴", "🧵", "🧶", "🧷", "🧸", "🧹", "🧺", "🧻", "🧼", "🧽", "🧾", "🧿", "🩰", "🩱", "🩲", "🩳", "🩴", "🩵", "🩶", "🩷", "🩸", "🩹", "🩺", "🩻", "🩼", "🪀", "🪁", "🪂", "🪃", "🪄", "🪅", "🪆", "🪇", "🪈", "🪐", "🪑", "🪒", "🪓", "🪔", "🪕", "🪖", "🪗", "🪘", "🪙", "🪚", "🪛", "🪜", "🪝", "🪞", "🪟", "🪠", "🪡", "🪢", "🪣", "🪤", "🪥", "🪦", "🪧", "🪨", "🪩", "🪪", "🪫", "🪬", "🪭", "🪮", "🪯", "🪰", "🪱", "🪲", "🪳", "🪴", "🪵", "🪶", "🪷", "🪸", "🪹", "🪺", "🪻", "🪼", "🪽", "🪿", "🫀", "🫁", "🫂", "🫃", "🫃🏻", "🫃🏼", "🫃🏽", "🫃🏾", "🫃🏿", "🫄", "🫄🏻", "🫄🏼", "🫄🏽", "🫄🏾", "🫄🏿", "🫅", "🫅🏻", "🫅🏼", "🫅🏽", "🫅🏾", "🫅🏿", "🫎", "🫏", "🫐", "🫑", "🫒", "🫓", "🫔", "🫕", "🫖", "🫗", "🫘", "🫙", "🫚", "🫛", "🫠", "🫡", "🫢", "🫣", "🫤", "🫥", "🫦", "🫧", "🫨", "🫰", "🫰🏻", "🫰🏼", "🫰🏽", "🫰🏾", "🫰🏿", "🫱", "🫱🏻", "🫱🏻‍🫲🏼", "🫱🏻‍🫲🏽", "🫱🏻‍🫲🏾", "🫱🏻‍🫲🏿", "🫱🏼", "🫱🏼‍🫲🏻", "🫱🏼‍🫲🏽", "🫱🏼‍🫲🏾", "🫱🏼‍🫲🏿", "🫱🏽", "🫱🏽‍🫲🏻", "🫱🏽‍🫲🏼", "🫱🏽‍🫲🏾", "🫱🏽‍🫲🏿", "🫱🏾", "🫱🏾‍🫲🏻", "🫱🏾‍🫲🏼", "🫱🏾‍🫲🏽", "🫱🏾‍🫲🏿", "🫱🏿", "🫱🏿‍🫲🏻", "🫱🏿‍🫲🏼", "🫱🏿‍🫲🏽", "🫱🏿‍🫲🏾", "🫲", "🫲🏻", "🫲🏼", "🫲🏽", "🫲🏾", "🫲🏿", "🫳", "🫳🏻", "🫳🏼", "🫳🏽", "🫳🏾", "🫳🏿", "🫴", "🫴🏻", "🫴🏼", "🫴🏽", "🫴🏾", "🫴🏿", "🫵", "🫵🏻", "🫵🏼", "🫵🏽", "🫵🏾", "🫵🏿", "🫶", "🫶🏻", "🫶🏼", "🫶🏽", "🫶🏾", "🫶🏿", "🫷", "🫷🏻", "🫷🏼", "🫷🏽", "🫷🏾", "🫷🏿", "🫸", "🫸🏻", "🫸🏼", "🫸🏽", "🫸🏾", "🫸🏿"}
```

Comment:
hm weird it looks like some emojis are in the list twice, e.g. "⛓‍💥"

Reply:
Yep, emojis are weird. But all of these have unique hex codes. You can see it if you convert both versions into bytes or hex code.